packages/ui-extensions/src/surfaces/admin/shouldRender.ts

```diff
@@ -0,0 +1,6 @@
+import {createShouldRenderRegistrationFunction} from '../../utilities/shouldRenderRegistration';
+
+import type {ExtensionTargets} from './extension-targets';
```

Comment:
We should probably create new types for shouldRender since the api will look different

Reply:
Ah totally! 😅